<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DijkstraClass</class>
 <widget class="QMainWindow" name="DijkstraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dijkstra算法演示</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,0">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>672</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>方正粗黑宋简体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>过程</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="table"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_nextstep">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>下一步</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_finish">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>结束</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_edt"/>
   <addaction name="action_load"/>
   <addaction name="action_start"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_edt">
   <property name="icon">
    <iconset resource="Dijkstra.qrc">
     <normaloff>:/Dijkstra/Resources/edit_24px_1208057_easyicon.net.png</normaloff>:/Dijkstra/Resources/edit_24px_1208057_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>编辑图</string>
   </property>
  </action>
  <action name="action_start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Dijkstra.qrc">
     <normaloff>:/Dijkstra/Resources/flag_24.15444015444px_1227697_easyicon.net.png</normaloff>:/Dijkstra/Resources/flag_24.15444015444px_1227697_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>选择起点</string>
   </property>
  </action>
  <action name="action_load">
   <property name="icon">
    <iconset resource="Dijkstra.qrc">
     <normaloff>:/Dijkstra/Resources/load.png</normaloff>:/Dijkstra/Resources/load.png</iconset>
   </property>
   <property name="text">
    <string>加载图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Dijkstra.qrc"/>
 </resources>
 <connections/>
</ui>
